web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig

```diff
@@ -0,0 +1,23 @@
+<div  class="recent-posts">
+  <h4>
+    <span>
+      {% trans %}
+        RECENT POSTS
+      {% endtrans %}
```

Comment:
It doesn't matter much but for the future
For simple string you can use {{ 'Recent Posts'|t }}
It's called twig filter

A trans block is generally for more complicated things.